frontend/src/views/authentication/Login.jsx

```diff
@@ -17,10 +17,17 @@ const Login = () => {
         formData.append('user_name', username);
         formData.append('password', password);
 
-        const getUser = async() => {
+        const getUser = async () => {
             const user = await postFormData("http://127.0.0.1:8000/", "login", formData)
-            setCurrentUser(user);
-            navigate("/");
+            
+            if (!(typeof user === 'undefined')) {

```

Comment:
`undefined` as string?

Reply:
<typeof user> returns a string literal hence why it is compared with 'undefined'